lib/logstash/filters/ruby.rb

```diff
@@ -44,6 +44,9 @@ class LogStash::Filters::Ruby < LogStash::Filters::Base
   # Tag to add to events that cause an exception in the script filter
   config :tag_on_exception, :type => :string, :default => "_rubyexception"
 
+  # Flag for add exception message to tag_on_exception
+  config :enable_exception_message, :type => :boolean, :default => false
```

Comment:
This setting could be named better, if a user looks at the docs they can be told what it does, but if they take over maintenance of a config with this in it, its not very obvious what the setting is designed to do.
I suggest `tag_with_exception_message`.
Change this and I think we can merge this PR.